integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/model/TestFixture.groovy

```diff
@@ -49,6 +49,7 @@ class TestFixture {
   ]
 
   public static final defaultRatingsU = [
```

Comment:
As the concept is now changed, I suggest renaming this method to `defaultRatingsClosed`